### PR TITLE
add toast component

### PR DESCRIPTION
**Pull request by ntassone**
image

### CODE REVIEW

File: src/components/_toast.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  display: flex;
+  flex-direction: row;
+  padding: 9px 16px 7px;
+  color: color.$color-grayLightest;
+  background-color: color.$color-grayDarker;
+  border-radius: 4px;
+  box-shadow: shadow.$shadow-far;
+  @include typography.Manifold__Typography__Label;
+
+  > svg {
```
Comment (dangodev) [inline src/components/_toast.scss:17]: 👍

File: stories/toast.stories.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+    ${comment.modifier}
+    
+    .Manifold__Banner--Error {
+        @include mercury.Manifold__Banner--Error;
+    }
+
+  `,
+      }}
+    />
```
Comment (dangodev) [inline stories/toast.stories.js:131]: nit: the indentation is a bit off, and missing a final `}`:

Screen Shot 2020-04-17 at 10 24 16

Comment (dangodev) [reply stories/toast.stories.js:131]: But dealing with this sucks, eh? Should probably hook this up to Prettier or something

Comment (ntassone) [reply stories/toast.stories.js:130]: No big deal. But yeah, if we didn't have to think about it that'd be even better!

**Pull request merged**